### PR TITLE
Vicinity/existing issues and exclude rules

### PR DESCRIPTION
**Pull request by t-8ch**
This is a reworked version of #24, splitting it into one commit per feature for easier review and applying to the current codebase.

### CODE REVIEW

**Comment by t-8ch** [comment]
Cc @christiangalsterer @ghenzler

File: src/main/java/org/sonar/plugins/stash/issue/StashDiffReport.java [hunk 1]
```diff
@@ -17,6 +19,7 @@
 
   private List<StashDiff> diffs;
 
+  public static int VICINITY_RANGE_NONE = 0;
```
Comment (re-schneider) [inline src/main/java/org/sonar/plugins/stash/issue/StashDiffReport.java]: MINOR Rename this field "VICINITY_RANGE_NONE" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS3008)
MINOR Make this "public static VICINITY_RANGE_NONE" field final [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1444)
MINOR Make VICINITY_RANGE_NONE a static final constant or non-public and provide accessors if needed. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)

File: src/main/java/org/sonar/plugins/stash/StashRequestFacade.java [hunk 2]
```diff
@@ -24,6 +25,7 @@
 import org.sonar.plugins.stash.issue.CoverageIssuesReport;
 import org.sonar.plugins.stash.issue.Issue;
 import org.sonar.plugins.stash.issue.MarkdownPrinter;
+import org.sonar.plugins.stash.issue.SonarQubeIssue;
```
Comment (re-schneider) [inline src/main/java/org/sonar/plugins/stash/StashRequestFacade.java]: MINOR Remove this unused import 'org.sonar.plugins.stash.issue.SonarQubeIssue'. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by re-schneider** [comment]
SonarQube analysis reported 4 issues
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Comment by re-schneider** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Pull request merged**